scss/heroNotificationBox.scss

```diff
@@ -0,0 +1,14 @@
+.heroNotificationBox {
```

Comment:
I imagine the white text on this background might be difficult to read. So, the design has a mostly transparent white filter on the notif box. I'd like to have that here or to see proof that this is easily readable

Reply:
Addressed in recent commit

Reply:
@NerajM rmb to `resolve conversation` if you have fixed the suggested changes